vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/features/metadata/day/DayMetadata.kt

```diff
@@ -0,0 +1,85 @@
+package org.vitrivr.engine.core.features.metadata.day

```

Comment:
I guess it is debatable if we want these things to remain in the core or if we'd rather have a module for date/time operations and data types. I could imagine that it would be generally useful to have something like this if it is designed to be sufficiently generalizable.

Reply:
I fully agree. More so, in my opinion, I would feel more confident in having `created` and `updated` as fields on the `FileSourceMetadata` to showcase the use of date time metadata than this very specific _lifelog_ usecase in the project at all.

As the PR says: There will be a cleanup afterwards.